src/Polly.Core/README.md

```diff
@@ -1,71 +1,68 @@
-# Introduction
+# Polly V8 API Documentation
 
-The Polly V8 API exposes unified and non-allocating resilience API that is described in the sections below.
+The Polly V8 API offers a unified, non-allocating resilience API, detailed in the sections below.
 
-## Core API
+## Introduction
 
-At the heart of Polly V8 is the [ResiliencePipeline](ResiliencePipeline.cs) class that is responsible for execution of user code. It's one class that handles all Polly V7 scenarios:
+At the core of Polly V8 is the [`ResiliencePipeline`](ResiliencePipeline.cs) class, responsible for executing user code. This class handles all scenarios covered in Polly V7, such as:
 
 - `ISyncPolicy`
 - `IAsyncPolicy`
 - `ISyncPolicy<T>`
 - `IAsyncPolicy<T>`
 
-``` csharp
+```csharp
 public abstract class ResiliencePipeline
 {
     public void Execute(Action callback);
 
     public TResult Execute<TResult>(Func<TResult> callback);
-
+    
     public Task ExecuteAsync(Func<CancellationToken, Task> callback, CancellationToken cancellationToken = default);
-
+    
     public Task<TResult> ExecuteAsync(Func<CancellationToken, Task<TResult>> callback, CancellationToken cancellationToken = default);
-
-    public ValueTask ExecuteValueTaskAsync(Func<CancellationToken, ValueTask> callback, CancellationToken cancellationToken = default);
-
-    public ValueTask<TResult> ExecuteValueTaskAsync(Func<CancellationToken, ValueTask<TResult>> callback, CancellationToken cancellationToken = default);
     
-    // omitted for simplicity
+    public ValueTask ExecuteAsync(Func<CancellationToken, ValueTask> callback, CancellationToken cancellationToken = default);
+    
+    public ValueTask<TResult> ExecuteAsync(Func<CancellationToken, ValueTask<TResult>> callback, CancellationToken cancellationToken = default);
+    
+    // Other methods are omitted for simplicity
```

Comment:
The snippet renders with a horizontal scrollbar due to the length of some of the method declarations. Maybe for readability we should spread the parameters onto new lines like below?

```csharp
public ValueTask<TResult> ExecuteAsync(
    Func<CancellationToken, ValueTask<TResult>> callback,
    CancellationToken cancellationToken = default);
```